platform/binance/client.go

```diff
@@ -98,3 +99,55 @@ func (c Client) FetchTokens() (Tokens, error) {
 	c.Cache.Set("tokens", *result, cache.DefaultExpiration)
 	return *result, nil
 }
+
+func (c *Client) GetValidators() (validators ValidatorsResponse, err error) {
+	cachedResult, ok := c.Cache.Get("validators")
+	if ok {
+		return cachedResult.(ValidatorsResponse), nil
+	}
+	query := url.Values{
+		"status": {"bonded"},
+	}
+	result := new(ValidatorsResponse)
+	resp, err := req.Get(c.url+"/api/v1/staking/validators", query)
+	if err != nil {
+		return *result, err
+	}
+	if err := resp.ToJSON(&result); err != nil {
```

Comment:
use `https://github.com/trustwallet/blockatlas/blob/e90b7df9a3bb607b9791da377e313fe97a82d3f5/platform/kava/client.go#L41`

you don't need to have custom json parsing